docs/UserGuide.md

```diff
@@ -80,6 +80,10 @@ Examples of usage:
 ### Add Deadline to a Manga: `schedule`
 Adds a deadline to an existing manga. `-b` stands for by-date (in case you are wondering why it is not `-d`, which is used for deleting in other commands). 
 
+It should be noted that the deadline does not need to follow any set format.
+This allows for greater flexibility in setting deadlines (e.g. `20 Jan 2024`, `Tuesday`, `After Mum's bday`, `in 2 days`).
+It is up to the user to define what is valid.
+
```

Comment:
I suggest that you place this in a note under the command. See the other notes in the User Guide on how to do this.

It should look like this in Markdown:
![{A8C1F090-1B72-44D6-8382-483644BDEAE7}](https://github.com/user-attachments/assets/0ef859e5-be32-43c4-84c0-d0d0136dc62b)

Reply:
Ok, I will do this and commit the changes shortly.